tests/by-util/test_lsmem.rs

```diff
@@ -2,3 +2,11 @@
 //
 // For the full copyright and license information, please view the LICENSE
 // file that was distributed with this source code.
+
+use crate::common::util::TestScenario;
+
+#[test]
+#[ignore]
```

Comment:
I don't know why you added `#[ignore]` as the test should pass.
```suggestion
```

Reply:
removed